crates/cheatcodes/src/evm.rs

```diff
@@ -823,18 +838,24 @@ impl Cheatcode for broadcastRawTransactionCall {
         let tx = TxEnvelope::decode(&mut self.data.as_ref())
             .map_err(|err| fmt_err!("failed to decode RLP-encoded transaction: {err}"))?;
 
-        ccx.ecx.db.transact_from_tx(
+        let env = Env::from(
+            ccx.ecx.inner.inner.cfg.clone(),
+            ccx.ecx.inner.inner.block.clone(),
+            ccx.ecx.inner.inner.tx.clone(),
+        );
+        let mut journaled_state = ccx.ecx.inner.inner.journaled_state.clone();
```

Comment:
obviously not ideal, this is a temporary workaround until the double `&mut` issue has been resolved